subsys/bluetooth/host/audio/iso.c

```diff
@@ -207,6 +207,7 @@ void hci_le_cis_req(struct net_buf *buf)
 	bt_conn_unref(conn);
 
 	if (!iso) {
+		BT_ERR("Could not create and add ISO to conn %d", acl_handle);
```

Comment:
Minor: `acl_handle` is unsigned, so `%u` would be more appropriate (would be a real bug if it was full 32-bits, but it's not)

Reply:
Ah, I see there's a similar bug with cis_handle, so probably a copy-paste thing

Reply:
And another instance with acl_handle as well

Reply:
Fixed :)

Reply:
@Thalley I think you forgot to push?

Reply:
So I did :)